src/Components/Sidebar/Tab.tsx

```diff
@@ -0,0 +1,35 @@
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
+
+export interface TabProps {
+  /** 탭 이름 */
+  title: string;
+
+  /** 선택되었을 떼 열릴 탭 라우트 */
+  to?: string;
+}
+
+/** 사이드바의 여러 탭 중 하나를 표현하는 컴포넌트 */
+export const Tab = ({ title, to }: TabProps) => {
+  return <NavLink to={to}>{({ isActive }) => <Box $active={isActive}>{title}</Box>}</NavLink>;
+};
+
+const Box = styled.div<{
+  $active: boolean;
```

Comment:
`$`표시는 컴포넌트의 props와 구분하기 위해 사용하시는 건가요?!

Reply:
`$` 표시는 저번에 저희가 논의했던 스타일드 컴포넌트의 warning을 발생시키는 것을 막기 위해서 사용됩니다.

기본적으로 스타일에 사용되는 prop을 받을 때, `$`를 사용해 transient props로 정의할 것을 권장합니다.

> Transient props v5.1
> If you want to prevent props meant to be consumed by styled components from being passed to the underlying React node or rendered to the DOM element, you can prefix the prop name with a dollar sign ($), turning it into a transient prop.

출처: https://styled-components.com/docs/api#transient-props

Reply:
그렇군요!!. 참고하겠습니다.🙏